internal/git/git_test.go

```diff
@@ -119,6 +119,12 @@ func TestPathWithNameSpace(t *testing.T) {
 			expected:    "zaquestion/test",
 			expectedErr: "",
 		},
+		{
```

Comment:
Can you put a comment somewhere in here or in the `PathWithNamespace` function mentioning the docs?
Ref: https://docs.gitlab.com/ce/user/group/subgroups/

Reply:
I put it in `PathWithNamespace` doc. Is this a good place for it to be?